template/Pages/Home/index.js

```diff
@@ -1,40 +1,63 @@
-import React from 'react'
-import { useLanguage } from 'ordering-components'
-import AiOutlineAndroid from '@meronex/icons/ai/AiOutlineAndroid'
-import AiOutlineApple from '@meronex/icons/ai/AiOutlineApple'
+import React, { useEffect, useState } from 'react'
+import { useLanguage, useApi } from 'ordering-components'
 import { HomeHero } from '../../../src/components/HomeHero'
 import { useHistory } from 'react-router-dom'
 import { HelmetTags } from '../../components/HelmetTags'
+import Skeleton from 'react-loading-skeleton'
+import { NotFoundSource } from '../../../src/components/NotFoundSource'
 
 import {
   HomeContainer,
-  HomeSection,
-  HomeTitle,
-  StepsBlock,
-  ImageContent,
-  WrapTextContent,
-  TextContent,
-  Icons,
-  HomeFooter,
-  Line
+  SkeletonContainer,
+  SkeletonHeader,
+  SkeletonContent,
+  SkeletonInformation,
+  SkeletonSide
 } from './styles'
 
-import locationSvg from '../../../template/assets/location-icon.svg'
-import businessSvg from '../../../template/assets/store-icon.svg'
-import productSvg from '../../../template/assets/product-icon.svg'
-import deliverySvg from '../../../template/assets/delivery-icon.svg'
-import burgerImage from '../../../template/assets/burger-image.png'
-import socialMedia from '../../../template/assets/social-media.png'
-import foodBg from '../../../template/assets/food-bg.png'
-
 export const HomePage = (props) => {
   const [, t] = useLanguage()
   const history = useHistory()
+  const [body, setBody] = useState(null)
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState(null)
```

Comment:
@YisusJe  You can create an one object instead of 3 useStates, for example: 

`const [homeState, setHomeState] = useState({ body: null, loading: false, error: null })`